lib/src/mjpeg.dart

```diff
@@ -146,11 +156,12 @@ class _StreamManager {
   final Map<String, String> headers;
   final Client _httpClient;
   final MjpegPreprocessor _preprocessor;
+  final Function _mounted;
```

Comment:
can you enforce the real type please to prevent usage of _StreamManager with function that doesn't return a boolean that would make everything crash

Reply:
👍  Fixed in new commit